protocol-units/execution/opt-executor/src/executor/initialization.rs

```diff
@@ -70,30 +70,45 @@ impl Executor {
 		(genesis, test_validators)
 	}
 
-	pub fn bootstrap_empty_db(
+	pub fn maybe_bootstrap_empty_db(
```

Comment:
It would be good to document what "maybe" means here. Or clarify with naming `bootstrap_db_if_empty`?

Generally, fallible functions do not need to be prefixed with `try_` or `maybe_` unless it's needed to disambiguate from the infallible variant present alongside.

Reply:
I'll commit once this runner stops, so that I don't bog it down.